packages/server-boilerplate/src/orchestrator/api/ApiBuilder.ts

```diff
@@ -147,9 +173,20 @@ export class ApiBuilder {
     ...handlers: RequestHandler<Params<T>, ResBody<T>, ReqBody<T>, Query<T>>[]
   ) {
     if (this.verifyAuthMiddleware) {
-      this.app[method](path, this.attachSession, this.verifyAuthMiddleware, ...handlers);
+      this.app[method](
+        this.formatPath(path),
+        this.attachSession as RequestHandler<Params<T>, ResBody<T>, ReqBody<T>, Query<T>>,
```

Comment:
Using this `as RequestHandler<Params<T>, ResBody<T>, ReqBody<T>, Query<T>>` as lot since otherwise we get typescript errors. They were there before, but for some reason `server-boilerplate` compiles fine even when it has typescript errors...